manifests/script-lab-react-localhost.outlook.xml

```diff
@@ -727,6 +727,9 @@
         </bt:String>
       </bt:LongStrings>
     </Resources>
```

Comment:
Why is indentation weird in this manifest...? :)

Reply:
will push fix